src/stores/device/cell.ts

```
@@ -0,0 +1,279 @@
+  public min?: number;
+  public max?: number;
+  public step: number;
+  public order: number;
+
+  private _value: any = '';
+  private _readOnly: boolean | null = null;
+  private _name: string;
+  private _nameTranslations: NameTranslations = {};
+  private _enumValues: { [lang: string]: { name: string; value: any }[] } = {};
+  private _enumTranslations: EnumTranslations = {};
+  private _units: string = '';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)